### DISCUSSION TITLE
Line appears after holding checkmark

### DISCUSSION THREAD

**Discussion by mscpotts** [iSoron/uhabits#1700] [Help & FAQ]
Hello,

What does the line mean when you select the check mark, it colors the checkmark, but then you hold it again and it becomes a colored line? Holding it a third time removes the checkmark. What is the second option and can it be added to the faq page?

**Comment by hiqua** [comment]
Mmh yeah probably worth an explanation, see also https://github.com/iSoron/uhabits/discussions/1615

The dash means "skip" day (if they're enabled in your settings), meaning that it doesn't break a cycle if you use them.
